android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -232,6 +228,24 @@ boolean isRunningBinaryVersion() {
         return sIsRunningBinaryVersion;
     }
 
+    private boolean isPackageBundleLatest(JSONObject packageMetadata) {
+        Long binaryModifiedDateDuringPackageInstall = null;
+        String binaryModifiedDateDuringPackageInstallString = packageMetadata.optString(CodePushConstants.BINARY_MODIFIED_TIME_KEY, null);
+        if (binaryModifiedDateDuringPackageInstallString != null) {
+            binaryModifiedDateDuringPackageInstall = Long.parseLong(binaryModifiedDateDuringPackageInstallString);
+        }
```

Comment:
Aye, good note, and it can be moved away from `getJSBundleFileInternal` method probably